### PR TITLE
{lang}[fosscuda/2019a] SciPy-bundle v2019.03

### PR DESCRIPTION
**Pull request by akesandgren**
(created using `eb --new-pr`)
Depends on ~~#8063~~

### CODE REVIEW

**Comment by akesandgren** [comment]
Test report by @akesandgren
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
b-an03.hpc2n.umu.se - Linux ubuntu 16.04, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/aab9813b221cfc11e293475d5ac32d71 for a full test report.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
b-an03.hpc2n.umu.se - Linux ubuntu 16.04, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/fe21592793e12be583fcd085e52d484c for a full test report.

**Comment by boegel** [comment]
@akesandgren Looks like your `Python` was not installed with a `Python` easyblock that includes the changes added in https://github.com/easybuilders/easybuild-easyblocks/pull/1657 ?

**Comment by akesandgren** [comment]
It should have been. I used EB 3.9.0 and first installed it on my production, then made the PR and the test-report. But i'll clean out the python builds from the test-report setup and try again.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
b-an03.hpc2n.umu.se - Linux ubuntu 16.04, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/9f836e3e621b4c36215ae4fd71ece566 for a full test report.

**Comment by akesandgren** [comment]
Apparently something was wrong in my test-report setup...

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3100.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/fb8f8164d21756aa09133af5ce6582da for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2413.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/ccc7a50b11bc34f2bbd3d02fa14887de for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2093.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/3be00b181c23ea30bcdd6de928ea7ea2 for a full test report.

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
vera-c1 - Linux centos linux 7.6.1810, Intel Xeon Processor (Skylake), Python 2.7.5
See https://gist.github.com/4b8cb67d38772418752f2da0865e55ab for a full test report.

File: easybuild/easyconfigs/s/SciPy-bundle/SciPy-bundle-2019.03-fosscuda-2019a.eb [hunk 1]
```diff
@@ -0,0 +1,42 @@
+easyblock = 'PythonBundle'
+
+name = 'SciPy-bundle'
+version = '2019.03'
+
+homepage = 'http://python.org/'
+description = "Bundle of Python packages for scientific software"
+
+toolchain = {'name': 'fosscuda', 'version': '2019a'}
+toolchainopts = {'pic': True, 'lowopt': True}
```
Comment (boegel) [inline easybuild/easyconfigs/s/SciPy-bundle/SciPy-bundle-2019.03-fosscuda-2019a.eb]: I see no reason for the `lowopt` here, I think it slipped in with `SciPy-bundle-2019.03-foss-2019a.eb` by mistake (and I'll open a separate PR to drop it).

Comment (boegel) [reply easybuild/easyconfigs/s/SciPy-bundle/SciPy-bundle-2019.03-fosscuda-2019a.eb]: I was just retesting with `lowopt` removed from `SciPy-bundle-2019.03-foss-2019a.eb`, and it seems like the `scipy` test suite exhibits failures with at least one test segfaulting on Intel Skylake X when `lowopt` is not enabled... :-/

**Comment by boegel** [comment]
Going in, thanks @akesandgren!

**Pull request merged**